modin/data_management/factories/factories.py

```diff
@@ -60,152 +107,147 @@ def get_info(cls) -> FactoryInfo:
         )
 
     @classmethod
-    def prepare(cls):
-        """
-        Fills in .io_cls class attribute lazily
-        """
+    @doc(_doc_factory_prepare_method, engine_name="an underlying")
+    def prepare(cls):  # noqa: D102
         raise NotImplementedError("Subclasses of BaseFactory must implement prepare")
 
     @classmethod
-    def _from_pandas(cls, df):
+    def _from_pandas(cls, df):  # noqa
```

Comment:
`GL08: The object does not have a docstring`, this check can't be disabled with `noqa: GL08`, so just using empty `noqa`

Reply:
@anmyachev this looks weird, could you pls have a look?

@dchigarev you should just write a docstring, and this error would go away :)

Reply:
added docstrings

Reply:
We can use `noqa: GL08` after #3001 is merged.